src/client/src/api/fetchApiV2.js

```diff
@@ -422,54 +404,6 @@ export const getBoreholeGeometryDepthMasl = async (boreholeId, depthMD) => {
   return await fetchApiV2(`boreholegeometry/getDepthInMasl?boreholeId=${boreholeId}&depthMD=${depthMD}`, "GET");
 };
 
-export const getWaterIngress = async boreholeId => {
```

Comment:
Ursprünglich hatten wir mal abgemacht diese Funktionen alle hier in `fetchApiV2` zu plazieren. Mit diesem change verabschieden wir uns von diesem Grundsatz.

Reply:
Der Grundsatz gilt schon länger nicht mehr, siehe Workgroup.ts, User.ts, borehole.ts ... Ich glaube das haben Dani und Roswita mal (während meines Mutterschaftsurlaubs ?) eingeführt. fetchApiV2 ist einfach zu gross und unübersichtlich geworden.
Aber vielleicht macht es Sinn alle neuen .ts files auch im api folder abzulegen, statt in den hydrogeology folders (?)

Reply:
Ja, fetchApiV2 ist zu gross und unübersichtlich geworden. Ich hätte die Files, die kein UI enthalten und nur interfaces für API objekte und fetch methoden eher im src/client/src/api Ordner platziert.

Reply:
Ja finde ich gut! Liefere einen PR nach